packages/gatsby-cli/src/__tests__/index.js

```diff
@@ -57,19 +60,9 @@ describe(`error handling`, () => {
   })
 })
 
-describe(`deprecation warning`, () => {
-  it(`warns on Node < 10.13.0`, () => {
-    const { reporter } = setup(`v10.12.0`)
-
-    expect(reporter.warn).toHaveBeenCalledWith(
-      expect.stringContaining(`https://gatsby.dev/upgrading-node-js`)
-    )
-  })
-})
-
```

Comment:
given that you commented out warning part in code, should this one also be commented out (and not removed)? Just so we don't have rewrite it when we get to deprecating node 10 in the future?

Reply:
good idea